wnfs-wasm/tests/share.spec.ts

```diff
@@ -0,0 +1,88 @@
+///<reference path="server/index.d.ts"/>
+
+import { expect, test } from "@playwright/test";
+
+const url = "http://localhost:8085";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(url);
+});
+
+test.describe("Share", () => {
+  test("share and recieve share", async ({ page }) => {
+    const result = await page.evaluate(async () => {
+      const {
+        wnfs: {
+          PrivateForest,
+          SharePayload,
+          share,
+          createShareLabel,
+          receiveShare,
+        },
+        mock: {
+          MemoryBlockStore,
+          Rng,
+          createSharerDir,
+          createRecipientExchangeRoot,
+          ExchangeKey,
+        },
+      } = await window.setup();
+
+      // @ts-ignore
+      globalThis.ExchangeKey = ExchangeKey;
```

Comment:
Yeah. This goes on the pile of things I'm not happy about with wasm-bindgen's choice of not exposing an `exports = await init(imports)` kind of api :/